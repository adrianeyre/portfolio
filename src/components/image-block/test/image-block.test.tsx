import { shallow } from 'enzyme';

import IImageBlockProps from '../interface/image-block-props';
import ImageBlock from '../image-block';

describe('Image Block', () => {
	const defaultProps: IImageBlockProps = {
		data: undefined,
		screenWidth: 0,
	};
	
	it('Should render correctly', () => {
		const imageBlockComponent = shallow(<ImageBlock { ...defaultProps }/>);

		expect(imageBlockComponent).toMatchSnapshot();
	});
});
