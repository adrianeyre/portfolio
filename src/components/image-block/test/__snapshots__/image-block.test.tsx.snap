// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Block Should render correctly 1`] = `ShallowWrapper {}`;
